client/van_site_config_create.go

```diff
@@ -0,0 +1,65 @@
+package client
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/skupperproject/skupper/api/types"
+)
+
+func (cli *VanClient) VanSiteConfigCreate(ctx context.Context, spec types.VanSiteConfigSpec) (*types.VanSiteConfig, error) {
+	siteConfig := &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            "skupper-site",
+		},
+		Data: map[string]string{
+		},
+	}
+	if spec.SkupperName != "" {
```

Comment:
Would it be beneficial to populate siteConfig.Data with entire set of key/values? It might make it easier to make changes without having to go back an related to the site spec.

Reply:
I don't quite understand what you are proposing here, could you elaborate a little?

Reply:
Presently, certain key/values are populated based on the spec settings (e.g. edge, console,etc.). Proposal would be to include the complete set as either default value 
spec setting override. So in all cases, the config map would have:

name
edge
service-controller
service-sync
console
console-authentication
console-user
console-password
cluster-local

That way  the user knows what the keys are if/when they would like to modify config map directly.